controllers/account/ec2.go

```diff
@@ -142,6 +142,7 @@ func (r *AccountReconciler) InitializeRegion(
 
 	// If in fedramp, create vpc
 	if config.IsFedramp() {
+		reqLogger.Info("Attempting to clean region", "account", account.Name, "region", region)
```

Comment:
Maybe make this more distinct so we know it's the pre-init cleanup process
```suggestion
		reqLogger.Info("Performing region initialization pre-cleanup of resources, "account", account.Name, "region", region)
```